src/popup/container.js

```diff
@@ -66,6 +67,8 @@ class PopupContainer extends Component {
 
         // Tooltip Flag
         isTooltipEnabled: true,
+
+        unreadNotifCount: 0,
```

Comment:
Really encourage you (and everyone) not to alter this popup container anymore. It's already a big mess.  Even though these are relatively small additions, why not just create a separate container for this button - which handles everything needed - then just add a single line to the popup container to render that container?

Reply:
Shouldn't all this stuff be in Redux anyway?

Reply:
When we first did the popup, it started off being very simple (a few buttons + the search input) and we just wanted to get it out asap. Hence we never set things like redux up for the the popup script. It's a lot more featureful now, however we haven't had any in-between time to clean things up or do things like migrate to redux. Refactoring of the popup script's codebase is on the feature freeze airtable though :)